Analysis/IsolatedLeptonTagging/include/IsolatedLeptonTaggingWOYokeProcessor.h

```diff
@@ -0,0 +1,82 @@
+#ifndef IsolatedLeptonTaggingWOYokeProcessor_h
```

Comment:
I've considered that option, but it seems better to create a new one, which is just for temporary use before potential problem gets fixed. In the end, we don't want to provide that option to confuse users, and the original version can be directly used without any change.

Reply:
This problem is actually not so temporary. It occurred in the central production samples and these will not be re-processed. It means that people analyzing data will have to deal with it as soon as they analyze these particular samples. 

I still believe it should go in a separate option that that must be well documented. In principle we would like to avoid code duplication when possible.